README.md

```diff
@@ -75,15 +76,18 @@ async for chunk in client.tts("Hi, I'm Jennifer from Play. How can I help you to
 
 The `tts` method takes the following arguments:
 
-- `text`: The text to be converted to speech.
-    - a string or list of strings.
-- `options`: The options to use for the TTS request.
-    - a `TTSOptions` object (see below).
-- `voice_engine`: The voice engine to use for the TTS request.
-    - `Play3.0-mini-http` (default): Our latest multilingual model, streaming audio over HTTP. (NOTE that it is `Play` not `PlayHT` like previous voice engines)
-    - `Play3.0-mini-ws`: Our latest multilingual model, streaming audio over WebSockets. (NOTE that it is `Play` not `PlayHT` like previous voice engines)
-    - `Play3.0-mini-grpc`: Our latest multilingual model, streaming audio over gRPC.  Use this voice engine if you're using Play On-Prem. (NOTE that it is `Play` not `PlayHT` like previous voice engines)
+- `text`: The text to be converted to speech; a string or list of strings.
+- `options`: The options to use for the TTS request; a `TTSOptions` object (see below).
```

Comment:
This might be a nice touch
```suggestion
- `options`: The options to use for the TTS request; a `TTSOptions` object [(see below)](#ttsoptions).
```